docs/changelog/100392.yaml

```
@@ -0,0 +1,5 @@
+pr: 100392
+summary: Prevent resource over-subscription in assignment planner
```

In the public docs we call these assignments model allocations, so it's best that the release note uses these words.

```suggestion
summary: Prevent resource over-subscription in model allocation planner
```